fdbcli/fdbcli.actor.cpp

```diff
@@ -80,6 +81,7 @@ CSimpleOpt::SOption g_rgOptions[] = { { OPT_CONNFILE, "-C", SO_REQ_SEP },
 	                                  { OPT_DATABASE, "-d", SO_REQ_SEP },
 	                                  { OPT_TRACE, "--log", SO_NONE },
 	                                  { OPT_TRACE_DIR, "--log-dir", SO_REQ_SEP },
+	                                  { OPT_LOGGROUP, "--log-group", SO_REQ_SEP },
```

Comment:
I would prefer to stick with `log-group` since the other commands use the same style e.g. `log-dir`. That's inconsistent with the other binaries e.g. `fdbserver` uses `loggroup` which might be solved in another PR.

Reply:
I'm on board with this spelling, though I think if we're going to do it we should simultaneously update the other binaries (this could still be a separate PR). If we're going to make that a future TODO task, then we should probably favor consistency and update this one along with the others later.